app/src/main/java/org/oppia/app/utility/PackageManagerHelper.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.pm.PackageManager
+import javax.inject.Inject
+
+/** Extension functions for PackageManager. */
+@Inject
+lateinit var context: Context
```

Comment:
We aren't injecting anything, so this will always be null. Plus, we don't actually need this if we have access to PackageManager via extension.

Reply:
Ended up extending Context instead per our chat